models/cloud_svc_epg.go

```diff
@@ -0,0 +1,112 @@
+package models
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/container"
+)
+
+const (
+	RnCloudSvcEPg        = "cloudsvcepg-%s"
+	DnCloudSvcEPg        = "uni/tn-%s/cloudapp-%s/cloudsvcepg-%s"
+	ParentDnCloudSvcEPg  = "uni/tn-%s/cloudapp-%s"
+	CloudSvcEPgClassName = "cloudSvcEPg"
+)
+
+type CloudServiceEPg struct {
+	BaseAttributes
+	CloudServiceEPgAttributes
+}
+
+type CloudServiceEPgAttributes struct {
+	AccessType           string `json:",omitempty"`
+	Annotation           string `json:",omitempty"`
+	AzPrivateEndpoint    string `json:",omitempty"`
+	CustomSvcType        string `json:",omitempty"`
+	DeploymentType       string `json:",omitempty"`
+	FloodOnEncap         string `json:",omitempty"`
+	MatchT               string `json:",omitempty"`
+	Name                 string `json:",omitempty"`
+	NameAlias            string `json:",omitempty"`
+	PrefGrMemb           string `json:",omitempty"`
+	Prio                 string `json:",omitempty"`
+	CloudServiceEPg_type string `json:",omitempty"`
+}
+
+func NewCloudServiceEPg(cloudSvcEPgRn, parentDn, description string, cloudSvcEPgAttr CloudServiceEPgAttributes) *CloudServiceEPg {
+	dn := fmt.Sprintf("%s/%s", parentDn, cloudSvcEPgRn)
+	return &CloudServiceEPg{
+		BaseAttributes: BaseAttributes{
+			DistinguishedName: dn,
+			Description:       description,
+			Status:            "created, modified",
+			ClassName:         CloudSvcEPgClassName,
+			Rn:                cloudSvcEPgRn,
+		},
+		CloudServiceEPgAttributes: cloudSvcEPgAttr,
+	}
+}
+
+func (cloudSvcEPg *CloudServiceEPg) ToMap() (map[string]string, error) {
+	cloudSvcEPgMap, err := cloudSvcEPg.BaseAttributes.ToMap()
+	if err != nil {
+		return nil, err
+	}
+
+	A(cloudSvcEPgMap, "accessType", cloudSvcEPg.AccessType)
+	A(cloudSvcEPgMap, "annotation", cloudSvcEPg.Annotation)
+	A(cloudSvcEPgMap, "azPrivateEndpoint", cloudSvcEPg.AzPrivateEndpoint)
+	A(cloudSvcEPgMap, "customSvcType", cloudSvcEPg.CustomSvcType)
+	A(cloudSvcEPgMap, "deploymentType", cloudSvcEPg.DeploymentType)
+	A(cloudSvcEPgMap, "floodOnEncap", cloudSvcEPg.FloodOnEncap)
+	A(cloudSvcEPgMap, "matchT", cloudSvcEPg.MatchT)
+	A(cloudSvcEPgMap, "name", cloudSvcEPg.Name)
+	A(cloudSvcEPgMap, "nameAlias", cloudSvcEPg.NameAlias)
+	A(cloudSvcEPgMap, "prefGrMemb", cloudSvcEPg.PrefGrMemb)
+	A(cloudSvcEPgMap, "prio", cloudSvcEPg.Prio)
+	A(cloudSvcEPgMap, "CloudServiceEPg_type", cloudSvcEPg.CloudServiceEPg_type)
```

Comment:
```suggestion
	A(cloudSvcEPgMap, "type", cloudSvcEPg.CloudServiceEPg_type)
```

Is it CloudServiceEPg_type or type?

Reply:
you are right. it is "type".